hngprojects/hng_boilerplate_nestjs
# refactor: add more columns to job entity

### Summary

This PR refactors the `Job` entity by adding new columns to accommodate additional job-related details. The following columns have been added:

- `qualifications`: Array of qualifications required for the job.
- `key_responsibilities`: Array of key responsibilities associated with the job.
- `job_description`: Detailed description of the job role.
- `benefits`: Array of benefits provided with the job.
- `experience_level`: Required or preferred experience level for the job.

### Changes Made

- Added `qualifications` column (array of strings).
- Added `key_responsibilities` column (array of strings).
- Added `job_description` column (string).
- Added `benefits` column (array of strings).
- Added `experience_level` column (string).

### How to Test

1. **Verify API**: Check the API documentation to confirm that the new fields are properly documented.
2. **Test Entity**: Verify that the `Job` entity behaves as expected with the new columns.

### Additional Notes

- Make sure to update any relevant documentation and tests to cover the new fields.
- If there are existing entries in the database, consider how the new columns will be handled (e.g., setting default values).
